src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -1,12 +1,17 @@
 import {createMaterialTopTabNavigator, MaterialTopTabNavigationEventMap} from '@react-navigation/material-top-tabs';
 import {EventMapCore, NavigationState, ScreenListeners} from '@react-navigation/native';
 import React from 'react';
+import {Platform} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
 import {OnyxEntry} from 'react-native-onyx/lib/types';
 import Tab from '@userActions/Tab';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ChildrenProps from '@src/types/utils/ChildrenProps';
 
+const TabNavigatorAnimationEnabled = ['ios', 'android'].includes(Platform.OS);
```

Comment:
I am not sure if we are okay with `Platform` imports? We generally have [platform specific files](https://github.com/Expensify/App?tab=readme-ov-file#platform-specific-file-extensions).

Reply:
@stitesExpensify @mountiny if you can comment here?

Reply:
Yeah definitely, let's stick to conventions! I will change that accordingly to your the final decision 👍

Reply:
Agreed, we should not be checking platforms like that

Reply:
Yes thats against our guidelines

Reply:
Done ✅  @mananjadhav